From 813ef961c2c762978481ae132ee73a5a1bdbe41e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Wed, 1 Sep 2010 15:45:00 +0000
Subject: [PATCH 01/13] git-clone: use builtin.h to get gettext.h
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change builtin/clone.c to use builtin.h. The only other command in
'mainporcelain common' which doesn't include it yet is builtin/reset.c

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---

This is just here so that the series applies stand-alone to pu. It's
made redundant by my "[PATCH] builtin: use builtin.h for all builtin/
commands" patch, but Junio expressed some reservations about it, so
here's a more minimal version sufficient for this series.

 builtin/clone.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/builtin/clone.c b/builtin/clone.c
index 19ed640..994d527 100644
--- a/builtin/clone.c
+++ b/builtin/clone.c
@@ -8,7 +8,7 @@
  * Clone a repository into a different directory that does not yet exist.
  */
 
-#include "cache.h"
+#include "builtin.h"
 #include "parse-options.h"
 #include "fetch-pack.h"
 #include "refs.h"
@@ -16,7 +16,6 @@
 #include "tree-walk.h"
 #include "unpack-trees.h"
 #include "transport.h"
-#include "strbuf.h"
 #include "dir.h"
 #include "pack-refs.h"
 #include "sigchain.h"
-- 
1.7.2.2.579.g2183d

